exercises/pangram/canonical-data.json

```diff
@@ -12,57 +12,81 @@
       ],
       "cases": [
         {
-          "description": "sentence empty",
+          "description": "can handle an empty sentence",
           "property": "isPangram",
           "input": "",
           "expected": false
         },
         {
-          "description": "pangram with only lower case",
+          "description": "handles an undefined message as empty message",
           "property": "isPangram",
-          "input": "the quick brown fox jumps over the lazy dog",
+          "input": "null",
```

Comment:
The description is misleading and the input being the string "null" here is confusing.
What are you trying to test?

Reply:
I need to change this to ```null``` instead.
My reasoning is the same as I wrote here:
https://github.com/exercism/problem-specifications/pull/895#discussion_r139141823

For everyone else, here's the quote to prevent jumping around:
I see what you mean. 
But as far as I experienced it, test suits should check for edge cases. null checks are especially important (at least in almost all languages). 
In every educational piece of material the advice is not to only check for usual inputs, but for all possible (equivalent kinds of) inputs. null checks, empty checks and others are always on top of the example lists given. Just take any arbitrary course about QA and this will be part of it.

I think this is especially important, because we are in an educational context. The users should learn from the getgo what it means to write (arguably good) tests (first), because it is still a major problem, even upon seasoned programmers. 
From my own business experience I know that new programmers are completely lost when it comes to writing tests. But also experienced programmers fall in the traps of "I'll do it later", "This is a trivial method", "I can't test this", "There are too many dependencies" etc. Just take any talk or tutorial about writing tests or even TDD and you will see the same old counter-arguments or questions again and again from the comments or the audience. This has a reason: *Writing good tests is hard! Very hard to be exact and it is a skill that needs exercise.* And where better to begin than right from the start?